instrumentation/base/lib/opentelemetry/instrumentation/base.rb

```diff
@@ -32,7 +32,7 @@ module Instrumentation
     #
     #         # if the target library is present, is it compatible?
```

Comment:
From the conversation in today's SIG about places we can write down design decisions made, one place I can think of for this particular decision on preferences around determining an instrumented gem's version is the doc comment above the `compatible do…end` example in instrumentation/base.

```suggestion
    #         # if the target library is present, is it compatible?
    #         # Common ways to retrieve a gem's version for comparison:
    #         # - ::Gemname::VERSION if the gem declares a version constant
    #         # - ::Gemname.version if the gem has provided a version method on the gem class
    #         # If a VERSION constant or version method are not available, as a fallback you can:
    #         # - Gem.loaded_specs['gemname']&.version - attempt to retrieve the gemspec from the gems currently loaded and use its version
```